Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo()
+        {
+            const int maxEntries = 3;
+
+            for (int i = 1; i < maxEntries; i++)
+            {
+                Console.Clear();
+
+                RegisterNewPersonData(i);
+            }
+        }
+
+        static void RegisterNewPersonData(int entry)
+        {
+            Console.WriteLine($"Enter the following information for person #{entry}.");
+
+            string firstName = Checks.PromptString("Firstname: ");
+
+            string lastName = Checks.PromptString("Lastname: ");
+
+            int age = Checks.PromptInt("Age:");
+
+            float weight = Checks.PromptFloat("Weight in kilograms:");
+
+            float height = Checks.PromptFloat("Height in meters:");
+
+            var bmi = Checks.CalculateBmi(weight, height);
+
+            Summarize(firstName, lastName, age, weight, height, bmi);
+
+            PromptNewPerson(entry);
```

Comment:
I think this line should be placed after line 16 instead. Think of the "do only one thing" principle. Here your function does more than it should.